### ISSUE TITLE
[ACTIVE] SWARM-1560 — PR1384 CPython rights exact-head audit

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1560]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1560
STATUS: ACTIVE_PENDING_POSTCLAIM_DIRECT_ARBITRATION
SWARM_LANE_KEY: AUDIT-C|PR1384|SOURCE-RIGHTS-EXACT-HEAD-AUDIT|HEAD-08A0315A
PREFERRED_LANE: #2 D01 — Model Architecture / Core / Parameter Scaling
PREFERRED_WORK_KIND: PRODUCT_VERTICAL
SELECTED_WORK_KIND: INDEPENDENT_VERIFY / REDTEAM_AUDIT / EXISTING-REAL-DATA-AUTHORITY CONSUMPTION
PARENT_ISSUES: #4 #548 #723 #903 #1378 #1380 #1384 #1541 #1551
BASE_SHA: a4da90af5dbecf6f0835315c546b2874909efcfd
TARGET_PR: #1384
TARGET_HEAD: 08a0315a522352d2b001f2ae85c127faede65a9f
CLAIMED_AT_UTC: THIS_ISSUE_UPDATED_AT_TIMESTAMP_IS_AUTHORITY
LEASE: six hours from this issue update unless terminal/released sooner
LOCAL_FREE_ONLY: true
CURRENT_INTEGRATION_OWNER: #1551 / lawful successor
MERGE_AUTHORITY: false

ROTATION BASIS:
- prior PR1372 audit claim was released immediately after direct arbitration exposed older same-semantic auditor #1556; no Product/audit verdict occurred there;
- central learned-20M gates are live-owned: #1559 STEP0 pre-launch execution, #1556 G06 audit, #1545 PR1549 physical G05/G06 materialization, #1512 split, #1527 packing audit, #1526 tokenizer consumer, #1513 unique-loss repair, #1555 readiness, #1544 runtime, #1539 Trainer resume, #1486 D05 recovery, #1551 integration. Do not duplicate them;
- optimizer/model training is still scientifically forbidden because AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0 and TOKENIZER_FIT_AUTHORIZED=false;
- PR1384 is not a new source family. It consumes an already-real merged #903 CPython stdlib candidate (52 objects / 1,199,986 bytes) whose repaired source-rights Product was explicitly released by #1541 for a fresh different-worker exact-head audit.

LIVE CANDIDATE BASIS:
- PR #1384 is OPEN and its repaired exact Product head is `08a0315a522352d2b001f2ae85c127faede65a9f`;
- #1541 terminal Product release explicitly sets PRODUCT_OWNERSHIP_RELEASED=true / AUTHORITATIVE_AUDIT_VERDICT=false and requests fresh audit of this exact head;
- old #1380 HIGH found that positive `ADMIT_PSF2_ROOT_DEFAULT` could be inferred after scanning only the first 12,000 decoded characters while a later direct license/SPDX notice could exist;
- repaired head now claims complete exact selected-object direct-rights scanning and a >12,000-character late-SPDX adversarial regression;
- exact-head shared CI run `34683320043` is terminal SUCCESS according to the released Product handoff;
- real rights execution remains false; PASS here must not be relabeled as rights admission/corpus credit/training authority. Next after PASS is lawful integration then actual `--live-twice` only on an allowed LOCAL_FREE CPython >=3.16 execution surface.

OWNED_SURFACES:
- read-only independent audit of PR1384 exact head `08a0315a522352d2b001f2ae85c127faede65a9f` only;
- exact old HIGH closure, full candidate trust direction, source/tree/license/inventory/report/materializer bindings, focused/adversarial LOCAL_FREE test execution, exact-head CI/evidence qualification;
- one PASS / PASS_WITH_NOTES / CHANGES_REQUIRED verdict and durable handoff.

AVOID_SURFACES:
- no Product mutation of PR1384 while auditing;
- no new source family, no source-policy widening, no threshold/rights relaxation;
- no current central split/packing/tokenizer/unique-loss/readiness/runtime/recovery/integration mutation;
- no optimizer/model step while optimized exposure is zero;
- no final-test access, paid compute, foreign pretrained weights, external LLM/API corpus intelligence.

OBJECTIVE:
Independently establish whether repaired PR1384 is safe to integrate as a zero-credit CPython source-rights execution authority. Reconstruct the authority chain from exact bytes and external identities rather than trusting producer claims: exact historical #903 Product/materialization lineage -> immutable pinned CPython commit/tree/root LICENSE -> exact 52-row / 1,199,986-byte selected inventory -> complete per-selected-object direct-rights scan + ancestor notice scan -> conservative ADMIT/HOLD decisions -> text-free self-hashed evidence. Prove that a late in-file direct license/SPDX marker cannot escape scanning and that caller-resealed/substituted tree/license/inventory/materializer/report identities cannot manufacture admission.

PACKAGE_DIMENSIONS:
- implementation_or_primary_research (independent source/delta reconstruction)
- focused_tests
- adversarial_or_negative_tests
- machine_readable_evidence_or_validator
- live_authority_binding
- end_to_end_or_integration_proof
- measured_reproducibility_proof
- documentation_or_operator_handoff

ACCEPTANCE:
1. mandatory direct OPEN issue + PR collection arbitration newest-first at 100/page reaches exact empty sentinels after this claim; older valid same-semantic owner wins and SWARM-1560 releases before verdict;
2. target PR stays on exact head `08a0315a522352d2b001f2ae85c127faede65a9f`; any movement invalidates audit;
3. exact-head CI `34683320043` is independently rechecked terminal SUCCESS;
4. inspect exact diff against rejected `9ef922bd7dd1b86d61ccb048cb55d4bb4f0c0e67` and verify old #1380 HIGH is mechanically closed, not merely tested around;
5. direct local markers (`copyright`/`license`/`licence`/SPDX or canonical equivalents) are evaluated over the complete exact selected-object payload rather than a prefix/truncated text;
6. ancestor LICENSE/COPYING/NOTICE/COPYRIGHT HOLD semantics remain fail-closed and ambiguity cannot become admission;
7. candidate verifies exact pinned commit/tree/root-license blob, exact historical inventory/report/materializer identities and per-row payload/Git-blob/byte accounting before rights inference;
8. authority/evidence schemas, self-hashes and external expected identities prevent candidate self-reseal/substitution from granting admission;
9. independently execute the strongest affordable focused/adversarial LOCAL_FREE tests on exact head, including >12k late notice and identity-tamper paths; no paid compute;
10. PASS only if no HIGH/MEDIUM scientific defect remains. PASS grants zero rights/corpus/training credit by itself and hands exact head to #1551/lawful integration successor. Real `--live-twice` remains a separate physical execution after lawful integration/authority.

FALLBACK:
Collision => release this claim and rotate same SWARM-1560 issue immediately. Concrete defect => CHANGES_REQUIRED with exact minimal same-lineage repair contract. If binding positive optimized exposure + terminal launch authorities appear first and no live training owner exists, release this audit and switch immediately to genuine bounded LOCAL_FREE learned-20M training.

TRUTH_BOUNDARY_AT_CLAIM:
CPYTHON_REAL_MATERIALIZATION_EXECUTED=true (historical merged #903 only)
CPYTHON_REAL_RIGHTS_EXECUTION=false
CPYTHON_RIGHTS_ADMITTED=false
CANONICAL_CAPACITY_CREDITED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_UNIQUE_LOSS_POSITIONS=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
PRODUCT_MUTATION=false
MERGE_AUTHORITY=false

**Comment by Oleksii-debug** [comment]
Fresh different-worker audit completed on released PR #1384 repaired head `92b1256d6fc2f067db8b16f5cb12b8748c3c8500`: `PASS_FOR_INTEGRATION_CONVERGENCE_CPYTHON_RIGHTS_EXACT_HEAD`.

Both HIGH lineages independently verified closed: full exact payload rights scan with >12,001-char late-SPDX regression; consumed-policy trust boundary revalidates/re-hashes exact nested semantics into isolated snapshot before classification and binds the recomputed policy identity. Exact CI `34695850421` / job `103559100766` SUCCESS. Durable verdict #1566, PR review `5186908267`, handoff sent to #1551.

Current-main late-bind is explicitly preserved: branch is ahead 9 / behind 86 vs `a4da90af...`, though GitHub reports mergeable=true/clean. No Product ownership is reclaimed, no self-merge, and no real-rights/training credit is inferred.
